core/arch/arm/mm/tee_pager.c

```diff
@@ -147,11 +148,7 @@ void tee_pager_get_stats(struct tee_pager_stats *stats)
 }
 #endif /* CFG_WITH_STATS */
 
-/*
- * Reference to translation table used to map the virtual memory range
- * covered by the pager.
- */
-static struct core_mmu_table_info tbl_info;
+struct core_mmu_table_info tee_pager_tbl_info;
```

Comment:
I've moved the comment to `tee_pager.h`, but I can keep it here too if you'd like.